experiments/conductor/cmd/runner/utilities.go

```diff
@@ -134,6 +134,49 @@ func cdRepoBranchDirBash(opts *RunnerOptions, subdir string, stdin io.WriteClose
 	return msg
 }
 
+func checkLocalChanges(ctx context.Context, branch Branch, workDir string, option string) {
+	log.Print("Checking uncommitted changes: git status --porcelain")
+	statusCmd := exec.CommandContext(ctx, "git", "status", "--porcelain")
+	statusCmd.Dir = workDir
+
+	results, err := execCommand(statusCmd)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if results.Stdout != "" {
+		currentBranchCmd := exec.CommandContext(ctx, "git", "rev-parse", "--abbrev-ref", "HEAD")
+		currentBranchCmd.Dir = workDir
+
+		currentBranchResult, err := execCommand(currentBranchCmd)
+		if err != nil {
+			log.Fatal(err)
+		}
+		switch option {
+		case handleLocalChangeOptionCleanUp:
+			if err := gitStash(ctx, workDir); err != nil {
```

Comment:
git stash has side effects. It will certainly get you back to clean. Not sure I want to be building up large numbers of uncommitted changes in the background. (Not blocking)